documentation/source/reference/molecule/atomtype.rst

```diff
@@ -119,4 +119,8 @@ Atom type       Description
 ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
 ``Cl``          chlorine atom with any local bond structure
 ``Cl1s``        chlorine atom with three lone pairs and zero to one single bonds
+*Iodine atom types*
+------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+``I``          iodine atom with any local bond structure
```

Comment:
It might not look important at first glance, but for this table to look good in our online documentation the spacings have to be precise. Note that the descriptions for Iodine aren't aligned with the other descriptions (one blank space need to be added in each line before the description)